pages/about/history.vue

```diff
@@ -14,165 +14,183 @@
                 </p>
             </template>
         </banner>
```

Comment:
banner: wait for https://github.com/pycontw/pycontw-frontend/pull/240

Reply:
#266 (refactored from #240) will update the banner without affecting this PR.